lib/codelog/cli.rb

```diff
@@ -4,6 +4,13 @@
 
 module Codelog
   class CLI < Thor
+    map ['--version', '-v'] => :__print_version
+
+    desc '[--version, -v]', 'prints current codelog version'
+    def __print_version
```

Comment:
I know, it is just a `puts` call, but would be nice to have an automated test to this command. It helps to keep the maintenability at a good grade and make it easier to detect regression bugs and issues.

Reply:
Nah, we don't even needed a command, so thats OK for me :|